client/js/components/fragments/Map.js

```diff
@@ -0,0 +1,80 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router';
+import GoogleMapsLoader from 'google-maps';
+
+class Map extends Component {
+  constructor () {
+    super();
+    this.map;
```

Comment:
note: check the part that talk about `getInitialState` here http://www.tamas.io/react-with-es6/, in ES6 this is handled in `constructor` instead.